Doc/library/configparser.rst

```diff
@@ -1421,6 +1426,13 @@ Exceptions
 
    .. versionadded:: 3.13
 
+.. exception:: UnnamedSectionDisabledError
+
+   Exception raised when attempting to use the unnamed section without enabling
+   it.
```

Comment:
```suggestion
   Exception raised when attempting to use the
   :const:`UNNAMED_SECTION` without enabling it.
```

Reply:
I didn't explicitly talked about `UNNAMED_SECTION` because it doesn't need to be explicitly refereed when parsing a file, except for allowing it with `enable_unnamed_section=True`.

Reply:
I'll await resolution of this concern before merging. @picnixz let us know if pslacerda's explanation satisfies your concern.

Reply:
I am fine with that. What matters to me is that people understand why such exception could be raised in general. I would have mentioned the constant so that you have a clickable link but it's also ok without it (it could make the docs slightly more readable without the link).